data/lang/module-combat/en.json

```diff
@@ -0,0 +1,346 @@
+   },
+   "PAYMENT_LOCATION" : {
+       "description" : "",
+       "message" : "Payment location:"
+   },
+   "POLICE_1" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "Police detectives have discovered that a group of pirates try to establish a stronghold on {area} in the {system} system ({sectorx}, {sectory}, {sectorz}), {dist} ly. This must not happen. The local authorities pay {cash} for a mission to {area} as preparation of a police operation."
+   },
+   "POLICE_2" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "The {org} will pay {cash} for a combat mission in the {system} system ({sectorx}, {sectory}, {sectorz}). We are short on pilots and ships. So this will be an opportunity for a loyal citizen to make money. The target is a small pirate base on {area}. You need hyperdrive propellant for {dist} ly and additional fuel for the return route."
+   },
+   "POLICE_3" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "Our markets were flooded by contrabant. The source has been traced back to a hidden base at {area} in the {system} system ({sectorx}, {sectory}, {sectorz}). Only a distance of {dist} ly. Because of the reduced police budget the local authorities would like to outsource this mission and have allotted {cash} credits for this operation."
+   },
+   "POLICE_IT_MUST_BE_COMPLETED_BY" : {
+      "description" : "",
+      "message" : "It's important that the mission is completed by {date}, because more hostile ships are on there way."
+   },
+   "POLICE_LAND_THERE" : {
+      "description" : "",
+      "message" : "Just land at a station with a {org} office."
+   },
+   "HARVESTING_1" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "The {org} will pay {cash} for a {mission} mission in the {system} system ({sectorx}, {sectory}, {sectorz}). We have reason to believe that a ruthless market competitor is responsible for the disapperance of our freighters at {area}. Distance to the target area is {dist} ly."
+   },
+   "HARVESTING_2" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "Our market competitor is seriously claiming he has ownership of {area} in the {system} system ({sectorx}, {sectory}, {sectorz}). The {org} totally disagree and is willing to pay {cash} for a combat mission at {area}. {area} is in a distance of {dist} ly."
+   },
+   "HARVESTING_3" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "Please help! We received a distress call from one of our freighters orbiting {area} in the {system} system ({sectorx}, {sectory}, {sectorz}). The call was interrupted, so we fear the worst. Distance to {system} is {dist} ly. The pay is {cash}."
+   },
+   "HARVESTING_IT_MUST_BE_COMPLETED_BY" : {
+      "description" : "",
+      "message" : "It's important that the mission is completed by {date}. Please keep shutm about this operation."
```

Comment:
> shutm

shtum/schtum?

Reply:
He probably meant keep shut, I would change to keep quiet